src/common/components/preferences/index.tsx

```diff
@@ -98,6 +101,34 @@ export class Preferences extends BaseComponent<Props, State> {
     success(_t("preferences.updated"));
   };
 
+  showSelfVoteChanged = (e: React.ChangeEvent<typeof FormControl & HTMLInputElement>) => {
+    const { setShowSelfVote } = this.props;
+    const { value } = e.target;
+    console.log(value);
+    const parsedValue = JSON.parse(value);
+    setShowSelfVote(parsedValue);
+    success(_t("preferences.updated"));
+  };
+
+  showRewardSplitChanged = (e: React.ChangeEvent<typeof FormControl & HTMLInputElement>) => {
+    const { setShowRewardSplit } = this.props;
+    const { value } = e.target;
+
+    setShowRewardSplit(JSON.parse(value));
+    success(_t("preferences.updated"));
+  };
+
+  lowRewardThresholdChanged = (e: React.ChangeEvent<typeof FormControl & HTMLInputElement>) => {
+    const { setLowRewardThreshold } = this.props;
+    const { value } = e.target;
+    try {
+      setLowRewardThreshold(JSON.parse(value));
+    } catch (e) {
+      console.log(value, "Couldn't get it done boss");
```

Comment:
We have a `error` function that will show an alert on the bottom right side of viewport. Please use it instead

Reply:
Error message not meaningful